include/ck/tensor_operation/gpu/device/impl/device_pool2d_fwd_nhwc_nhwc.hpp

```diff
@@ -22,6 +13,8 @@ template <typename InDataType,
           typename OutDataType,
           typename IndexDataType, // enable if OutputIndex == true
           typename ComputeDataType,
+          typename InLayout,
+          typename OutLayout,
```

Comment:
Why do you still need this when the NDHWC & NDHWC already determined ?

Reply:
b61339a36b8be395566f11c9e1583315273820c2